handbook/engineering/security/infrastructure/index.md

```diff
@@ -0,0 +1,135 @@
+# Security Infrastructure
+
+We maintain multiple flavors of infrastructure with various degrees of management. 
+
+
+
+## GCP Infrastructure Basics
```

Comment:
[We favor sentence cases](https://about.sourcegraph.com/handbook/communication/content_guidelines/actionable_language#headings-and-subheadings) :D

Reply:
Good catch, thanks so much!